transmute_core/object_serializers/attrs_serializer.py

```diff
@@ -0,0 +1,57 @@
+from .interface import ObjectSerializer
+from cattr import structure, unstructure
+from ..exceptions import SerializationException
+from cattr.vendor.typing import (
+    Optional,
+    List, MutableSequence, Sequence,
+    Tuple,
+    MutableSet, Set,
+    FrozenSet,
+    Dict, MutableMapping, Mapping,
+    Union,
+)
+import attr
+
+
+class AttrsSerializer(ObjectSerializer):
+    """
+    An ObjectSerializer which allows the serialization of
+    basic types and attrs classes.
+    """
+
+    def __init__(self):
+        pass
+
+    def load(self, model, value):
+        """
+        Converts unstructured data into structured data, recursively.
+        """
+        try:
+            res = structure(value, model)
+        except Exception as e:
```

Comment:
the specific exception should be caught here. a generic try/catch could catch other issues unrelated to the destructuring of the object.